### ISSUE TITLE
Requesting update the simplify function 

### ISSUE THREAD

**Issue by Strange369** [symengine/symengine.py#405]
Hi,
The simplify function here still calls sympy, which is really time consuming.
Since symengine has already implemented its own simplify function. 
Could you please update the simplify function here accordingly.
Thank you very much

**Comment by rikardn** [comment]
Thanks for reporting this. Although the `refine` and `simplify` are not nearly as mature as the ones in sympy, I think we should use the ones in symengine here. If the user wants to use sympy to simplify that would still be possible, by calling sympy directly. I guess this is legacy from a time when symengine didn't have any similar functionality.

**Comment by Goulustis** [comment]
How is the status of this enhancement?

**Comment by adrian-kriegel** [comment]
I would also appreciate this. What is currently holding this back? Doesn't the simplify function just need to be added in the wrapper?

**Comment by rikardn** [comment]
Yes, it needs to be added. This also requires the `Assumptions` class to be added somehow and this might not be as straight forward. Any ideas on how to wrap this for Python?
